ui/app/styles/components/tool-tip.scss

```diff
@@ -63,7 +63,9 @@
 }
 .ember-basic-dropdown-content--above.tool-tip {
   @include css-bottom-arrow(8px, $black, 1px, $black);
-  margin-top: -8px;
+  & {
```

Comment:
After adding the precompiled HDS css this was the only remaining deprecation warning
```sh
Deprecation Warning: Sass's behavior for declarations that appear after nested
rules will be changing to match the behavior specified by CSS in an upcoming
version. To keep the existing behavior, move the declaration above the nested
rule. To opt into the new behavior, wrap the declaration in `& {}`.

More info: https://sass-lang.com/d/mixed-decls

   ┌──> /var/folders/8_/0xphy4bj341dh508hct76byc0000gn/T/broccoli-4707xHYIWqO61Xe2/out-777-funnel_funnel_styles/app/styles/components/tool-tip.scss
66 │     margin-top: -8px;
   │     ^^^^^^^^^^^^^^^^ declaration
   ╵
   ┌──> /var/folders/8_/0xphy4bj341dh508hct76byc0000gn/T/broccoli-4707xHYIWqO61Xe2/out-777-funnel_funnel_styles/app/styles/utils/mixins.scss
58 │ ┌     &:after,
59 │ │     &:before {
60 │ │       left: auto;
61 │ │       right: calc(#{$left} + #{$left-offset} - #{$size});
62 │ │     }
   │ └─── nested rule
   ╵
    /var/folders/8_/0xphy4bj341dh508hct76byc0000gn/T/broccoli-4707xHYIWqO61Xe2/out-777-funnel_funnel_styles/app/styles/components/tool-tip.scss 66:3  @import
    /var/folders/8_/0xphy4bj341dh508hct76byc0000gn/T/broccoli-4707xHYIWqO61Xe2/out-777-funnel_funnel_styles/app/styles/core.scss 110:9                @import
    /var/folders/8_/0xphy4bj341dh508hct76byc0000gn/T/broccoli-4707xHYIWqO61Xe2/out-777-funnel_funnel_styles/app/styles/app.scss 9:9                   root stylesheet